sdk/servicebus/service-bus/test/utils/testUtils.ts

```diff
@@ -474,6 +486,88 @@ export function getNamespace(serviceBusConnectionString: string): string {
+  TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME = "TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME",
+  MANAGEMENT_QUEUE_1 = "MANAGEMENT_QUEUE_1",
+  MANAGEMENT_TOPIC_1 = "MANAGEMENT_TOPIC_1",
+  MANAGEMENT_SUBSCRIPTION_1 = "MANAGEMENT_SUBSCRIPTION_1",
+  MANAGEMENT_RULE_1 = "MANAGEMENT_RULE_1",
+  MANAGEMENT_QUEUE_2 = "MANAGEMENT_QUEUE_2",
+  MANAGEMENT_TOPIC_2 = "MANAGEMENT_TOPIC_2",
+  MANAGEMENT_SUBSCRIPTION_2 = "MANAGEMENT_SUBSCRIPTION_2",
+  MANAGEMENT_RULE_2 = "MANAGEMENT_RULE_2",
+  MANAGEMENT_NEW_ENTITY_1 = "MANAGEMENT_NEW_ENTITY_1",
+  MANAGEMENT_NEW_ENTITY_2 = "MANAGEMENT_NEW_ENTITY_2"
+}
+
+// Reference to cached entityNames that is unique per test run.
+let entityNames: any;
+
+/**
+ * Utility to return cached map of entity names,
+ * or create and return one from configured values if not existing.
+ */
+export function getEntityNameMap() {
+  console.log("THIS --> ", entityNames);
+  if (entityNames != undefined) {
+    return entityNames;
+  }
+
+  entityNames = {
+    [EntityNameKeys.QUEUE_NAME]: "partitioned-queue",
+    [EntityNameKeys.QUEUE_NAME_NO_PARTITION]: "unpartitioned-queue",
+    [EntityNameKeys.QUEUE_NAME_SESSION]: "partitioned-queue-sessions",
+    [EntityNameKeys.QUEUE_NAME_NO_PARTITION_SESSION]: "unpartitioned-queue-sessions",
+    [EntityNameKeys.TOPIC_NAME]: "partitioned-topic",
+    [EntityNameKeys.TOPIC_NAME_NO_PARTITION]: "unpartitioned-topic",
+    [EntityNameKeys.TOPIC_NAME_SESSION]: "partitioned-topic-sessions",
+    [EntityNameKeys.TOPIC_NAME_NO_PARTITION_SESSION]: "unpartitioned-topic-sessions",
+    [EntityNameKeys.SUBSCRIPTION_NAME]: "partitioned-topic-subscription",
+    [EntityNameKeys.SUBSCRIPTION_NAME_NO_PARTITION]: "unpartitioned-topic-subscription",
+    [EntityNameKeys.SUBSCRIPTION_NAME_SESSION]: "partitioned-topic-sessions-subscription",
+    [EntityNameKeys.SUBSCRIPTION_NAME_NO_PARTITION_SESSION]:
+      "unpartitioned-topic-sessions-subscription",
+    [EntityNameKeys.TOPIC_FILTER_NAME]: "topic-filter",
+    [EntityNameKeys.TOPIC_FILTER_SUBSCRIPTION_NAME]: "topic-filter-subscription",
+    [EntityNameKeys.TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME]: "topic-filter-default-subscription",
+    [EntityNameKeys.MANAGEMENT_QUEUE_1]: "management-queue-1",
+    [EntityNameKeys.MANAGEMENT_TOPIC_1]: "management-topic-1",
+    [EntityNameKeys.MANAGEMENT_SUBSCRIPTION_1]: "management-subscription-1",
+    [EntityNameKeys.MANAGEMENT_RULE_1]: "management-rule-1",
+    [EntityNameKeys.MANAGEMENT_QUEUE_2]: "management-queue-2",
+    [EntityNameKeys.MANAGEMENT_TOPIC_2]: "management-topic-2",
+    [EntityNameKeys.MANAGEMENT_SUBSCRIPTION_2]: "management-subscription-2",
+    [EntityNameKeys.MANAGEMENT_RULE_2]: "management-rule-2",
+    [EntityNameKeys.MANAGEMENT_NEW_ENTITY_1]: "management-new-entity-1",
+    [EntityNameKeys.MANAGEMENT_NEW_ENTITY_2]: "management-new-entity-2"
+  };
+
+  if (!isNode) {
+    Object.keys(entityNames).forEach((key) => {
+      entityNames[key] = entityNames[key] + "-browser";
```

Comment:
Once we have the arm template set up to use different Service Bus namespaces for node vs browser, there are 2 improvements we can make
- remove the need for `-browser` suffix here
- remove the need for `getEntityNameMap()`  altogether by renaming `EntityNameKeys` to `EntityNames` and using it directly.

Please create an issue for the above changes and have it blocked by the issue for the arm template.

Reply:
Logged #6738